### ISSUE TITLE
Startup Issue - Linux Mint Xfce4

### ISSUE THREAD

**Issue by zach-pagel** [diodechain/diode_drive_feedback#120]
**OS:** Linux Mint 20 Ulyana (Xfce desktop)
**Diode Drive Version:** 1.2.1

After installing a fresh Diode Drive and starting it, the related dDrive processes can be seen running, but no signup window or tray icon appears.

**Comment by zach-pagel** [comment]
There is no ~/.config/ddrive folder, so I think it's not getting that far.

**Comment by dominicletz** [comment]
Oh, then it must be a different issue. Try starting dDrive from command line then using `./dDrive start_iex` from the `$HOME/dDrive` directory it should print an error message

**Comment by zach-pagel** [comment]
Initially `Protocol 'inet_tcp': register/listen error: econnrefused`, followed by `Protocol 'inet_tcp': register/listen error: heart` every few seconds. Looks like maybe an `epmd` issue, do you know how I can debug that further?

**Comment by zach-pagel** [comment]
I just installed erlang with `apt install erlang` and now Diode Drive starts. Maybe this needs added to the docs?

**Comment by dominicletz** [comment]
erlang itself is not a dependency. But has the `apt install erlang` installed any subdependencies?

**Comment by zach-pagel** [comment]
Here are the deps that `apt install erlang` installed:
<br>
```
erlang-asn1 erlang-base erlang-common-test erlang-crypto erlang-debugger
erlang-dev erlang-dialyzer erlang-diameter erlang-edoc erlang-eldap
erlang-erl-docgen erlang-et erlang-eunit erlang-examples erlang-ftp
erlang-inets erlang-jinterface erlang-megaco erlang-mnesia erlang-mode
erlang-observer erlang-odbc erlang-os-mon erlang-parsetools
erlang-public-key erlang-reltool erlang-runtime-tools erlang-snmp erlang-src
erlang-ssh erlang-ssl erlang-syntax-tools erlang-tftp erlang-tools erlang-wx
erlang-xmerl javascript-common libglade2-0 libjpeg-turbo-progs libjs-jquery
libjs-jquery-metadata libjs-jquery-tablesorter libllvm10 libllvm11
libmediainfo0v5 libodbc1 libqt5x11extras5 libraw19 libsctp1 libtinyxml2-6a
libzen0v5 linux-headers-5.4.0-53 linux-headers-5.4.0-53-generic
linux-headers-5.4.0-73 linux-headers-5.4.0-73-generic linux-headers-5.4.0-74
linux-headers-5.4.0-74-generic linux-headers-5.4.0-77
linux-headers-5.4.0-77-generic linux-headers-5.4.0-80
linux-headers-5.4.0-80-generic linux-headers-5.4.0-81
linux-headers-5.4.0-81-generic linux-headers-5.4.0-84
linux-headers-5.4.0-84-generic linux-headers-5.4.0-88
linux-headers-5.4.0-88-generic linux-headers-5.4.0-89
linux-headers-5.4.0-89-generic linux-image-5.4.0-53-generic
linux-image-5.4.0-73-generic linux-image-5.4.0-74-generic
linux-image-5.4.0-77-generic linux-image-5.4.0-80-generic
linux-image-5.4.0-81-generic linux-image-5.4.0-84-generic
linux-image-5.4.0-88-generic linux-image-5.4.0-89-generic
linux-modules-5.4.0-53-generic linux-modules-5.4.0-73-generic
linux-modules-5.4.0-74-generic linux-modules-5.4.0-77-generic
linux-modules-5.4.0-80-generic linux-modules-5.4.0-81-generic
linux-modules-5.4.0-84-generic linux-modules-5.4.0-88-generic
linux-modules-5.4.0-89-generic linux-modules-extra-5.4.0-53-generic
linux-modules-extra-5.4.0-73-generic linux-modules-extra-5.4.0-74-generic
linux-modules-extra-5.4.0-77-generic linux-modules-extra-5.4.0-80-generic
linux-modules-extra-5.4.0-81-generic linux-modules-extra-5.4.0-84-generic
linux-modules-extra-5.4.0-88-generic linux-modules-extra-5.4.0-89-generic
xscreensaver-data
```

**Comment by zach-pagel** [comment]
Also, the Diode Drive tray icon is now gone. It appeared for a few moments at startup (after `apt install erlang`) but disappearred shortly after that. The Diode Drive window can still be started by starting the program twice.

**Comment by dominicletz** [comment]
Is this stable or nightly?

**Comment by zach-pagel** [comment]
Stable

**Comment by zach-pagel** [comment]
![os](https://user-images.githubusercontent.com/17348062/146083835-685379d2-ac10-4ef4-9a8f-923288839aef.png)

**Comment by dominicletz** [comment]
Sweet got something! Some notes here:

I downloaded the linux mint iso from https://www.linuxmint.com/edition.php?id=283 and started into a VM

* The initial start is fine and shows an icon (version v1.2.1)
* But after the auto-update to stable v1.3.9 the taskbar icon is gone
* Upgrading further to nightly, the app would not even start -- there was an issue with the username not yet being set during startup. I've fixed that bug on master just now. So given v1.3.10 it should be fine.

Note though that I could not find any connection to missing dependencies. The mint iso image in pristine state was running drive without issues...
